admiral/pkg/apis/admiral/v1/type.go

```diff
@@ -81,3 +81,31 @@ type RoutingPolicyList struct {
 
 	Items []RoutingPolicy `json:"items"`
 }
+
+// +genclient
+// +k8s:deepcopy-gen:interfaces=k8s.io/apimachinery/pkg/runtime.Object
+// +k8s:openapi-gen=true
+// +kubebuilder:printcolumn:name="Destination",type="string",JSONPath=`.spec.destination.identity`
+// +kubebuilder:printcolumn:name="Proxy",type="string",JSONPath=`.spec.proxy.identity`
+// +kubebuilder:resource:shortName=dp
+type DependencyProxy struct {
+	meta_v1.TypeMeta   `json:",inline"`
+	meta_v1.ObjectMeta `json:"metadata"`
+	Spec               model.DependencyProxy `json:"spec"`
+	Status             DependencyProxyStatus `json:"status"`
+}
+
+// DependencyProxyStatus is the status for a DependencyProxy resource
+type DependencyProxyStatus struct {
+	ClusterSynced int32  `json:"clustersSynced"`
```

Comment:
What is the logic to update this?

Reply:
@nirvanagit , since this was in the other types, so I just kept it as is.
@aattuluri , do you know if we need this or if this can be removed?

Reply:
@shriramsharma We should remove this given its doesnt make sense for this CRD.